### PR TITLE
Include Python snippets tab

### PR DESCRIPTION
**Pull request by shemogumbe**
## Overview

Brief description of what this PR does, and why it is needed.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -417,7 +417,7 @@ module.exports = function (webpackEnv) {
         maxChunks: 1
       }),
       new MonacoWebpackPlugin({
-        languages: ['json', 'javascript', 'java', 'csharp', 'html', 'powershell', 'go']
+        languages: ['json', 'javascript', 'java', 'csharp', 'html', 'powershell', 'go', 'python']
```
Comment (adhiambovivian) [inline config/webpack.config.js:420]: @shemogumbe we are now ordering the code snippets tab by alphabet.

File: src/app/views/query-response/snippets/Snippets.tsx [hunk 2]
```diff
@@ -28,7 +28,11 @@ function GetSnippets() {
     'JavaScript': {
       sdkDownloadLink: 'https://aka.ms/graphjssdk',
       sdkDocLink: 'https://aka.ms/sdk-doc'
-    }
+    },
```
[the remark this answers is not in the document]
Comment (adhiambovivian) [reply src/app/views/query-response/snippets/Snippets.tsx:31]: @Onokaev  let's add the step by step requirements or guidelines when onboarding new snippet to GE on our wiki eg the inofrmation on ordering by alphabet, SDK language documentation etc

File: config/webpack.config.js (hunk 1 above)
Comment (Onokaev) [reply config/webpack.config.js:420]: Hey @adhiambovivian. The ordering here does not affect the ordering on the UI

File: src/app/views/query-response/snippets/Snippets.tsx (hunk 2 above)
Comment (Onokaev) [inline src/app/views/query-response/snippets/Snippets.tsx:31]: I believe you are talking about the ordering here @adhiambovivian.
We automatically order the snippet tabs before they are rendered. So it doesn't matter where they are placed in this object

[the remark this answers is not in the document]
Comment (shemogumbe) [reply src/app/views/query-response/snippets/Snippets.tsx:31]: @Onokaev  I these are already there, 
`'Python': {
      sdkDownloadLink: 'https://aka.ms/msgraphpythonsdk',
      sdkDocLink: 'https://aka.ms/sdk-doc'
    },`
The aka link directs to [the docs](https://learn.microsoft.com/en-us/graph/sdks/sdks-overview)

**Review by Onokaev — DISMISSED** [review]

**Review by Onokaev — DISMISSED** [review]

**Review by Onokaev — APPROVED** [review]

**Pull request merged**